sacred/observers/tinydb_hashfs.py

```diff
@@ -58,7 +58,7 @@ def create_from(cls, db, fs, overwrite=None, root=None):
     def save(self):
         """Insert or update the current run entry."""
         if self.db_run_id:
-            self.runs.update(self.run_entry, eids=[self.db_run_id])
+            self.runs.update(self.run_entry, doc_ids=[self.db_run_id])
```

Comment:
Do you know when this changed? Does this break older installations? But even if it breaks old code, people should update to newer versions of these packages.

Reply:
According to the [changelog](https://tinydb.readthedocs.io/en/latest/changelog.html#breaking-changes) with Version 4.0.0.

Reply:
I believe this is fine